src/routes/Introduction/Introduction.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import classnames from 'classnames'
+import { t } from 'ttag'
+
+import { JRaisedButton } from 'components/base'
+
+import style from './introduction.m.scss'
+
+import { Slide } from './components/Slide'
```

Comment:
'./components/Slide' -> 'components'
And also we need task to update root alias (`$`) for webpack

Reply:
https://jibrelnetwork.atlassian.net/browse/JWWEB-750

Reply:
oh, sorry didn't see that we are in routes dir :)